integrations/telegram

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+import sys
+import json
+import requests
+import logging
+import urllib
+import urllib2
+from requests.auth import HTTPBasicAuth
+reload(sys)
+sys.setdefaultencoding('utf8')
+BLACK_LIST= ["sca","vulnerability-detector"]
```

Comment:
This `BLACK_LIST` and `BLACK_RULE` should not be added to the final version of the script. This is customization for a particular use case but in the default one, it should be tuned with the available filters in the integrations defined [here](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/integration.html#options).